src/events/ready.ts

```diff
@@ -1,10 +1,15 @@
 import { REST } from "@discordjs/rest";
 import { Routes } from "discord-api-types/v9";
 import {
+<<<<<<< Updated upstream
```

Comment:
More merge conflicts here

Reply:
@Juilzze resolved this issue in latest commit.